packages/types/src/events/assistant.ts

```diff
@@ -0,0 +1,29 @@
+export interface AssistantThreadStartedEvent {
```

Comment:
@filmaj These are used in the implementation for Bolt, but figured they should go in now. Let me know if you prefer these to be separate!